### DISCUSSION TITLE
n-gon-upgraded stuff

### DISCUSSION THREAD

**Discussion by 3xionDev** [landgreen/n-gon#346] [General]
This is a discussion space to discuss n-gon-upgraded outside of the issues tab, in order to not crowd landgreen with new issues unrelated to the game. Suggest features, notify me of bugs, or just talk about the mod in general.

**Comment by AnnonymousNerd87** [comment]
I just use my email now so it doesn't change much for me.

Love your mod btw

**Comment by smallbigman** [comment]
you should add new gun tech on wave and mines

**Reply by cubingpenguin** [reply]
i think that's too vague to really make a change...

**Comment by TechnoGuru123698741** [comment]
if it is possible, could you make some keybindings for tech "unified field theory: when paused you can click to change your field. 2x frequency for field tech" to be able to change the field without pausing, please?

**Comment by 3xionDev** [comment]
<img width="386" height="278" alt="image" src="https://github.com/user-attachments/assets/4ab46dec-9414-4ba0-8c92-c33e4d12da51" />
<img width="829" height="410" alt="image" src="https://github.com/user-attachments/assets/41032a08-77f9-483a-980c-93bd3bff87f3" />
<img width="829" height="410" alt="image" src="https://github.com/user-attachments/assets/c5c78482-3b9f-4a77-a9b2-58666f83a318" />

we are so back

**Reply by R3d5t0n3GUY** [reply]
We never left 😎

**Reply by smallbigman** [reply]
Peak

**Comment by jtopdawg14** [comment]
since the thread is alive again, are you going to take the new scource code from n-gon to keep n-gon upgraded up to date with unmodded n-gon? because that's one of the main reasons I don't play any n-gon mods.

**Reply by 3xionDev** [reply]
I'm about 38 merge conflicts behind the main branch but yes I do plan on merging them back together at some point when I get the motivation to do so. I unfortunately might have to change some current main game mechanics to keep my modded items, but if I do it right the mod should just feel like a natural extension of the main game. in the meantime try playing the mod, its just oldish n-gon with some fun additions

**Comment by R3d5t0n3GUY** [comment]
I have some ideas for the **energy condenser** field:

- Coupling effect: either increased max health, or larger heal powerups
- Field effect: ➡️⬆️⬅️⬇️⬇️ (if energy is above ~85% of max, set it to 1% in order to to full heal.)
- If using the **energy condenser**, selecting _mass-energy equivalence_ resets your field to **field emitter**. If using _mass-energy_, _unified field theory_ skips over **energy condenser**. If the player manages to have both **energy condenser** and _mass-energy_ simultaneously, the field behaves like **field emitter** (i.e. neither consumes energy to heal the player, nor can activate the full-heal key combo)

**Comment by rgb-44865** [comment]
Thoughts on this?
<img width="848" height="110" alt="image" src="https://github.com/user-attachments/assets/dd9474fd-5b35-4235-bd44-7efcb53b7040" />
Convalescence is unavailable if you have both mass-energy equivalence and time crystals

**Reply by 3xionDev** [reply]
see: accelerated mitosis
<img width="333" height="82" alt="image" src="https://github.com/user-attachments/assets/8541247c-53ee-43d0-8392-b5b5ef70c761" />
